game/sketch.js

```diff
@@ -175,11 +181,23 @@ function touchStarted() {
 function touchEnded() {
   controller.endSwipe(mouseX, mouseY, now());
   control();
+  delay(500).then(saveGame);
 
   // Prevent the default touch action.
   return false;
 }
 
+function saveGame() {
+
+  var progress = {
+    game: board.tiles,
+    score: board.score
+  };
+  localStorage.setItem('progress', JSON.stringify(progress));
+
+  console.log(progress)
+}
```

Comment:
Above function saves the current progress of the game, and loads it into LocalStorage as a JSON object.